### PR TITLE
Fix Pathing Nodes

### PR DESCRIPTION
**Pull request by NathanDuPont**
Fixed the HolonomicPursuit class, and updated the SetCurrentPose logic to handle pathing correctly

### CODE REVIEW

File: src/v5_hal/firmware/src/nodes/auton_nodes/AutonManagerNode.cpp [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
-    m_programming_skills_auton = new ProgrammingSkillsAuton(drive_node, odometry_node, conveyor_node, intake_node, inertial_sensor_node);
-    selected_auton = m_programming_skills_auton;
+    m_test_path_auton = new TestPathAuton(drive_node, odometry_node);
+    selected_auton = m_test_path_auton;
 }
 
 void AutonManagerNode::initialize() {
     PathManager::GetInstance()->LoadPathsFile("/usd/path.json");
-    selected_auton->AutonInit();
+    // selected_auton->AutonInit();
```
Comment (AndrewKempen) [inline src/v5_hal/firmware/src/nodes/auton_nodes/AutonManagerNode.cpp:11]: delete this

File: src/v5_hal/firmware/src/pathing/PathManager.cpp [hunk 2]
```diff
@@ -37,7 +37,7 @@ bool PathManager::LoadPaths(json loadedJson) {
             for (auto point : pathJson["points"]) {
                 Vector2d linear_velocity(point["vx"], point["vy"]);
                 float time = point["time"];
-                float rotational_velocity = point["omega"];
+                float rotational_velocity = point["omega"]; //TODO toRadians?
```
Comment (AndrewKempen) [inline src/v5_hal/firmware/src/pathing/PathManager.cpp:40]: No need todo

**Review by AndrewKempen — DISMISSED** [review]
Just a few comments to remove

**Review by Joseph5062A — APPROVED** [review]
👍

**Pull request merged**